extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -61,20 +60,17 @@ export class VisibilityModel {
     /** @private {?number} */
     this.repeatInterval_ = null;
```

Comment:
do we still need this?

Reply:
I believe that we still need this value, but its name is no longer appropriate. It is now just min(totalTimeMin, continuousTimeMin) and has nothing to do with the value of "repeat" anymore. I'll change it to something better, but am open to suggestions...

Reply:
Renamed to "refreshInterval" since it's ultimately used to set a timeout for calling this.refresh_()

Reply:
https://github.com/ampproject/amphtml/commit/733095904d7de4d6a39960d93e9d16c285ed6f65#diff-61c963752cc0a61e2ec5e1561dc0466cR69

this was the PR where this interval got introduced. looking at the line that I highlighted, the whole thing can be removed because it only functions when `reset` is set to a number.

Reply:
but please use example page to do some sanity check to see if the `repeat` keyword works as intended.

Reply:
Ok, cut more. All automated tests pass (though of course I just removed some that were no longer relevant). http://localhost:8000/test/manual/amp-analytics-visible-repeat.amp.html also looks right (including locally changing true to false and verifying expected results then, but not checking that in).